cmd/sync.go

```diff
@@ -461,7 +512,14 @@ type cookedSyncCmdArgs struct {
 
 	dryrunMode bool
 
-	cfdMode CFDModeFlags
+	// Change file detection Mode. Valid Enums are TargetCompare, Ctime, CtimeMtime.
+	// TargetCompare - Default sync comparsion where target enumerated for each file. It's least optimized, but gurantee no data loss.
+	// Ctime - CTime used to detect change files/folder. Should be used where mtime not reliable.
+	// CtimeMtime - Both CTime      and MTime used to detect changed files/folder. It's most efficient in all of the cfdModes.
+	cfdMode common.CFDMode
+
+	// In case of metadata change only, shall we transfer whole file or only metadata. This flag governs that.
+	metaDataOnlySync bool
```

Comment:
again, refer to the comments in rawSyncCmdArgs.
Or do you think we should add comments to cookedSyncCmdArgs  and refer all other places to here?
let's discuss.

Reply:
cookedSyncCmdArgs will be better.